src/harmony/Alert/Alert.module.css

```diff
@@ -0,0 +1,45 @@
+.Alert {
+    display: flex;
+    align-items: center;
+    justify-content: start;
+    border-radius: var(--radius-m);
+    padding: var(--gap-sm) var(--gap-sm);
+    gap: var(--gap-s);
+    width: 22.25rem;
+}
+
+.Alert_default {
+    color: var(--gray-900);
+    background: var(--gray-900);
+    z-index: 1;
+    color: var(--text-primary);
+}
+
+.Alert_danger {
+    color: var(--danger-300);
+    background: var(--danger-900);
+    z-index: 1;
+}
+
+.Alert_warning {
+    color: var(--warn-300);
+    background: var(--warn-900);
+    z-index: 1;
+}
+
+.Text {
+    font-size: var(--font-size-s);
+    line-height: var(--line-height-s);
+}
+
+.Text_default {
+    color: var(--gray-300);
```

Comment:
You are using same style for Alert container

Reply:
fixed